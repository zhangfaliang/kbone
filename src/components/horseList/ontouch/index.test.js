import toJson from 'enzyme-to-json';
import { render, mount } from 'enzyme';
import Carousel from './index';
// import moment from 'moment';
// import { getTimeFromCTSDateString } from '@/utils/getTimeFromCTSDateString';

describe('Horizontal', () => {
  const data = {
    item: () => <div>1213212</div>,
    data: [{ k: 1 }, { k: 2 }],
    id: '12311',
    numSrcoll: 0,
  };
  it('赛马场列表容器', () => {
    expect.assertions(1);
    const wrapper = render(<Carousel {...data} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('ontouch', () => {
    // expect.assertions(1);
    const wrapper = mount(<Carousel {...data} {...{ numSrcoll: 2 }} />);
    wrapper.setProps({
      numSrcoll: 3,
    });
    // expect(toJson(wrapper)).toMatchSnapshot();
  });
});
