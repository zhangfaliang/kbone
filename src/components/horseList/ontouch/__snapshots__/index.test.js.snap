// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Horizontal 赛马场列表容器 1`] = `
<div
  class="srcoll"
  id="12311"
>
  <div
    class="onTouchBody"
  >
    <div>
      <div>
        1213212
      </div>
    </div>
    <div>
      <div>
        1213212
      </div>
    </div>
  </div>
</div>
`;
